### PR TITLE
Enable pulp_manage_rsync boolean for rsync tests

### PR DESCRIPTION
**Pull request by Ichimonji10**
See:

* https://pulp.plan.io/issues/2199
* http://docs.pulpproject.org/plugins/pulp_rpm/tech-reference/iso-rsync-distributor.html#configuration

### CODE REVIEW

**Comment by Ichimonji10** [comment]
This should fix several test failures.

**Comment by Ichimonji10** [comment]
Tested against Pulp 2.10.2 on RHEL 6, RHEL 7 and Fedora 24, and Pulp 2.11.1 on Fedora 24.

File: pulp_smash/tests/rpm/api_v2/test_rsync_distributor.py [hunk 1]
```diff
@@ -92,13 +92,23 @@ def setUpModule():  # pylint:disable=invalid-name
...
+        cmd.insert(0, 'sudo')
+    cli.Client(cfg).run(cmd)
 
 
 def tearDownModule():  # pylint:disable=invalid-name
     """Delete orphan content units."""
-    api.Client(config.get_config()).delete(ORPHANS_PATH)
+    cfg = config.get_config()
+    api.Client(cfg).delete(ORPHANS_PATH)
+    cmd = 'semanage boolean --modify --off pulp_manage_rsync'.split()
```
Comment (elyezer) [inline pulp_smash/tests/rpm/api_v2/test_rsync_distributor.py]: Would not worth getting this logic as a helper function? The helper function can receive a boolean indicating to turn on and off and the config object.

This is not a blocker.

**Comment by Ichimonji10** [comment]
Good suggestion. Updated.

**Pull request merged**